test/unit_test/systems/python_environment/test_python_environment_system.py

```diff
@@ -91,7 +91,7 @@ def test_evaluate_accuracy(self, mock_compute_accuracy):
         expected_res = local_system.evaluate_model(model, [metric], DEFAULT_CPU_ACCELERATOR)["accuracy-accuracy_score"]
 
         # execute
-        actual_res = self.system.evaluate_accuracy(model, metric)[AccuracySubType.ACCURACY_SCORE]
+        actual_res = self.system.evaluate_model(model, [metric], DEFAULT_CPU_ACCELERATOR)["accuracy-accuracy_score"]
```

Comment:
@jambayk , I will let you decide the change. the change LGTM except the test case change.